src/nft/components/explore/Banner.tsx

```diff
@@ -77,6 +85,8 @@ const Banner = () => {
     }
   )
 
+  const collections = data?.filter((collection) => !EXCLUDED_COLLECTIONS.includes(collection.address))
```

Comment:
Also set a slice to prevent 6 from showing if punks isn't top 5

Reply:
Good catch!